storage-service/src/main/kotlin/com/icerockdev/service/storage/s3/policy/builder/StatementBuilder.kt

```diff
@@ -0,0 +1,41 @@
+package com.icerockdev.service.storage.s3.policy.builder
+
+import com.icerockdev.service.storage.s3.policy.dto.ActionEnum
+import com.icerockdev.service.storage.s3.policy.dto.EffectEnum
+import com.icerockdev.service.storage.s3.policy.dto.Principal
+import com.icerockdev.service.storage.s3.policy.dto.Statement
+import java.lang.Exception
+
+class StatementBuilder {
+    var effect: EffectEnum? = null
```

Comment:
effect not nullable in Statement, why is nullable in this class?

Reply:
To check the builder data

Reply:
need check it in `build()` dunction

Reply:
Done